hawk/app/lib/hawk/secure_cookies.rb

```diff
@@ -17,7 +17,8 @@ def call(env)
           next if cookie.blank?
           next if cookie =~ /;\s*secure/i
 
-          cookie << '; Secure ; HttpOnly'
+          cookie << '; Secure'
+          cookie << '; HttpOnly' if ENV['HAWK_COOKIE_HTTP_ONLY'] == 'true'
```

Comment:
I want to make it configurable **mostly** because of the comment 9 in the bsc#1198647 

_"Not sure if this is a permanent change you would want to make but do you feel this would better be entered in as a feature request"_ 

I've made all cookies HttpOnly in #254, but I think now, maybe it's better to make them HttpOnly only if the user wants it.

Reply:
My view is that this is something that should stay internal to the application because it is a mere implementation detail. All Hawk cookies that don't need to be accessed from JS should be possible to safely mark as `HttpOnly`. I cannot see when this would create a problem for users and one would want to disable it?

Reply:
> My view is that this is something that should stay internal to the application because it is a mere implementation detail. All Hawk cookies that don't need to be accessed from JS should be possible to safely mark as `HttpOnly`. I cannot see when this would create a problem for users and one would want to disable it?

This is convincing. It was exactly my reasoning when doing the #254. I will wait for the answer in the bugzilla and then likely close the PR.